test/runtests_parser.py

```diff
@@ -93,6 +105,19 @@ def clean(self):
             for f in files:
                 if "model-sets" not in f:
                     depends_on.append(Test.filename_to_test_id(f))
+
+            files = Parser.get_values_of_vwarg(self.vw_command, "--cache_file")
+            for f in files:
+                # If the file has not been registered as an output
+                if not Test.has_output(f):
+                    continue
```

Comment:
this if should never be executed bc of line 99. /I think/

Reply:
Good catch. Removed.